pkg/rancher-desktop/main/mainmenu.ts

```diff
@@ -1,7 +1,17 @@
 import Electron, { Menu, MenuItem, MenuItemConstructorOptions, shell } from 'electron';
 
+import { getVersion, parseDocsVersion } from '@pkg/utils/version';
 import { openMain } from '@pkg/window';
 
+const baseUrl = `https://docs.rancherdesktop.io`;
+
+const versionedDocsUrl = async() => {
+  const version = await getVersion();
+  const parsed = parseDocsVersion(version);
+
+  return `${ baseUrl }/${ parsed }`;
+};
```

Comment:
Any particular reason this isn't `async function versionedDocsUrl()` instead? (Maybe there's an important but non-obvious one I'm missing…)